src/openssl_wrap.cpp

```diff
@@ -614,6 +614,61 @@ time_point _asn1TimeToTimePoint(const ASN1_TIME *time)
     return epoch + (24h * days) + (1s * seconds);
 }
 
+/**
+ * @brief Convert an ASN1_TIME instance to a time_t
+ * @param [in] time An ASN1 encoded time object
+ * @return A time_t object with the same time
+ */
+time_t _asn1TimeToTimeT(const ASN1_TIME *time)
```

Comment:
This function introduces a lot of code duplication with _asn1TimeToTimepoint. I don't really like this. 
Couldn't we refactor the common parts into another function that is then instrumented to either create time_points or time_t?

Reply:
Done.